cmd/macoffice/generate.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server/vulnerabilities/macoffice"
+)
+
+func panicif(err error) {
```

Comment:
What is the the behaviour when we panic?
Do we restart the whole collection job set? 
Do we log it or know that it failed?

Reply:
The whole collection job will fail and an alert will be raised on Slack. The error details should be available in the GH action logs, see https://github.com/fleetdm/nvd/actions/runs/3814308964/jobs/6488503433#step:5:27 for an example

Reply:
Understood. Thanks.